app/views/components/_accordion.html.erb

```diff
@@ -1,8 +1,9 @@
 <%
   sections ||= false
+  controls_on_left ||= false
 %>
 <% if sections %>
-  <div data-module="accordion" class="app-c-accordion js-hidden">
+  <div data-module="accordion" class="app-c-accordion js-hidden <% if controls_on_left %>app-c-accordion--controls-on-left<% end %>">
```

Comment:
(Again, I'd personally opt for `<%= "app-c-accordion--controls-on-left" if controls_on_left %>`